oci/oci_transport.go

```diff
@@ -47,12 +54,24 @@ func ParseReference(reference string) (types.ImageReference, error) {
 			return nil, fmt.Errorf("Invalid tag %s", tag)
 		}
 	}
-	return NewReference(dir, tag), nil
+	return NewReference(dir, tag)
 }
 
 // NewReference returns an OCI reference for a directory and a tag.
-func NewReference(dir, tag string) types.ImageReference {
-	return ociReference{dir: dir, tag: tag}
+//
+// We do not expose an API supplying the resolvedDir; we could, but recomputing it
+// is generally cheap enough that we prefer being confident about the properties of resolvedDir.
+func NewReference(dir, tag string) (types.ImageReference, error) {
+	resolved, err := private.ResolvePathToFullyExplicit(dir)
+	if err != nil {
+		return nil, err
+	}
+	// This is necessary to prevent directory paths returned by PolicyConfigurationNamespaces
+	// from being ambiguous with values of PolicyConfigurationIdentity.
+	if strings.Contains(resolved, ":") {
+		return nil, fmt.Errorf("Invalid OCI reference %s:%s: path %s contains a colon", dir, tag, resolved)
```

Comment:
acceptable, I don't see why not. OCI_dest right now is kind of the same as dir - OCI image transportation and distribution isn't even yet defined so I treat oci: as dir: and this lgtm